packages/collector/test/tracing/protocols/apollo_subgraph/test.js

```diff
@@ -18,16 +17,12 @@ const globalAgent = require('../../../globalAgent');
 const agentControls = globalAgent.instance;
 
 const mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
-
-mochaSuiteFn('tracing/apollo-federation', function () {
+mochaSuiteFn('tracing gateway with apollo-subgraph', function () {
   this.timeout(config.getTestTimeout() * 2);
-
   globalAgent.setUpCleanUpHooks();
-
   const allControls = startAllProcesses();
 
-  [false, true].forEach(withError => registerQuerySuite.bind(this)(allControls, { withError }));
-  // registerQuerySuite.bind(this)({ withError: false });
+  [true, false].forEach(withError => registerQuerySuite.bind(this)(allControls, { withError }));
```

Comment:
line 90 needs to be updated I guess

>   // Not using ProcessControls.setUpHooks(...) here because it starts all processes simultaneously, but for Apollo
>  // Federation it is necessary to start the processes sequentially and in order.